src/codeu/chat/server/Controller.java

```diff
@@ -34,9 +40,18 @@
   private final Model model;
   private final Uuid.Generator uuidGenerator;
 
-  public Controller(Uuid serverId, Model model) {
+  public PrintWriter output;
+  public static boolean writeToLog;
```

Comment:
public members must be final; otherwise, use a private member with a public getter.